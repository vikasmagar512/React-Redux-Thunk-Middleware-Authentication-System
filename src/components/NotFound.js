import React from 'react';
import {Link} from 'react-router';

const NotFound = () =>
  <div className="container">
      <p>Seems like you took the wrong way.</p>
      <Link to="/">Go to Homepage </Link>
  </div> ;

export default NotFound;