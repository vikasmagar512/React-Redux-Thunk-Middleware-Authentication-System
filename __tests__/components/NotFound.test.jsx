import {Link,IndexLink} from 'react-router';
import React,{Component} from 'react';
import createReactClass from  'create-react-class';
import PropTypes from 'prop-types';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import toJson from 'enzyme-to-json';
import { shallow } from 'enzyme';
import NotFound from '../../src/components/NotFound';


describe('NotFound', () => {
  it('should match its empty snapshot', () => {
    const tree = renderer.create(
      <NotFound/>
     ).toJSON();
    expect(tree).toMatchSnapshot();
  });
/*    it('renders without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(<App/>, div);
    });
*/   
}); 