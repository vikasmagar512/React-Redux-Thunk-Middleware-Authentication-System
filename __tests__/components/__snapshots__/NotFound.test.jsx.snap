// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound should match its empty snapshot 1`] = `
<div
  className="container"
>
  <p>
    Seems like you took the wrong way.
  </p>
  <a
    onClick={[Function]}
    style={Object {}}
  >
    Go to Homepage 
  </a>
</div>
`;
